### ISSUE TITLE
why docgen doesn't produce text inside doc comments?

### ISSUE THREAD

**Issue by sergorl** [OpenZeppelin/solidity-docgen#83]
I have the contract below:
```sol
pragma solidity ^0.5.6;

/// @title A simulator for trees
/// @author Larry A. Gardner
/// @notice You can use this contract for only the most basic simulation
/// @dev All function calls are currently implemented without side effects
contract Tree {
    /// @author Mary A. Botanist
    /// @notice Calculate tree age in years, rounded up, for live trees
    /// @dev The Alexandr N. Tetearing algorithm could increase precision
    /// @param rings The number of rings from dendrochronological sample
    /// @return age in years, rounded up for partial years
    function age(uint256 rings) external pure returns (uint256) {
        return rings + 1;
    }
}

```
Command ```solidity-docgen [ -i <input-dir> ] [ -o <output-dir> ] ``` gives me .md file, but I don't see the doc comments? Why It doesn't happen?

**Comment by frangio** [comment]
Hi @sergorl, thank you for reporting. Let me look into it.

**Comment by frangio** [comment]
There was an error in the NatSpec parsing code. Fixed in 008ca8a and released as 0.3.12.

Thanks! Let me know if you run into any other issues.
